uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/document-list.js

```diff
@@ -113,18 +113,15 @@ define([
             };
 
             $scope.filterParams = {
-                contactId: null,
-                documentStatus: [],
-                userRole: $state.params.userRole || {
-                    field: null,
-                    isEqual: null
-                },
-                dateRange: {
-                    from: null,
-                    until: null
-                },
-                due: null,
-                assignmentType: []
+              contactId: null,
```

Comment:
Isn't a service an overkill for a small json object? Not to mention that there is at least one difference between one `filterParams` and the other, which in the future could become two or more, undermining the reason for the service to exist in the first place. Could you provide an example of what you have in mind?